docker-plugin/src/test/java/com/nirima/jenkins/plugins/docker/ws/MappedFsWorkspaceBrowserTest.java

```diff
@@ -0,0 +1,84 @@
+package com.nirima.jenkins.plugins.docker.ws;
+
+import com.nirima.jenkins.plugins.docker.action.DockerBuildAction;
+import hudson.FilePath;
+import hudson.model.Job;
+import hudson.model.Run;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.apache.commons.lang.RandomStringUtils.randomAlphabetic;
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.startsWith;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+/**
+ * User: lanwen
```

Comment:
Please put right javadoc `@author lanwen` or better your first and last names. Date and time imho redundant.